### ISSUE TITLE
Issue with 1Password Extension in TestFlight Builds 

### ISSUE THREAD

**Issue by pres** [AgileBits/onepassword-app-extension#359]
I'm seeing a weird issue that I have no explanation for.

I have the 1password extension in my app since quite a while now.
I recently updated to `1.8.4` and to `cocoapods 1.1.0` (from `0.39`).

The issue is, that the 1password buttons in the app are not showing when the app is installed using TestFlight.

- Building locally works fine.
- Installing the build from a CI server works fine.
- Using the App Store build works fine (although the App Store version is from before the update)
- TestFlight no luck – the button is just not there …

Except the App Store version everything else used the same commit.
Same OS (10.1) and same device. 

Any idea what the issue could be?
Or is it perfectly normal that the extension does not work using TestFlight builds?

**Comment by gks** [comment]
You could possibly add some logging to see what result you get back for the `[[OnePasswordExtension sharedExtension] isAppExtensionAvailable]` call. It returns a bool. My guess is that for some reason you're getting back an incorrect value here (not sure why in this case) or possibly how you're handling the resulting action is behaving differently with TestFlight builds than those run from Xcode. 

I assume you're using the same stable version of 1Password for this test? Or are you also using a TestFlight build?

**Comment by pres** [comment]
1Password App is the latest version from the App Store.

**Comment by pres** [comment]
Ok … I fixed. 

It seems there was an issue the button image.
I used `[UIImage imageNamed:@"onepassword-button.png" inBundle:onepasswordExtensionResourcesBundle compatibleWithTraitCollection:nil];`

After your comment I added logging to sign up and registration.
And also changed signup to simply use `[UIImage imageNamed:@"onepassword-button"];` + copied the button in our Apps `xcassets`.

After creating a new TestFlight Build both signup and registration works again.
